src/components/Beers.js

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import BeerDetails from "./BeerDetails";
+
+function Beers() {
+  const [beers, setBeers] = useState([]);
+  const [fetching, setFetching] = useState(true);
+
+  useEffect(() => {
+    axios.get("https://ih-beers-api2.herokuapp.com/beers").then((response) => {
+      setBeers(response.data);
+      setFetching(false);
+    });
+  }, []);
+
+  return (
+    <>
+      <header>
+        <Link to="/">
+          <i>home</i>
+        </Link>
+      </header>
+
+        {fetching && <div></div>}
+        {beers.map((beer) => (
+          
+            <div key={beer._id}>
+              <img
+                style={{ height: "200px" }}
+                src={beer.image_url}
+                alt=""
+              ></img>
+                  <h5>
+                    <Link
+                      class="card-link"
+                      to={`/beers/${beer._id}`}
+                      element={<BeerDetails beer={beer} />}>
+                      {beer.name}
+                    </Link>
```

Comment:
In the app.js you define all the possible routes and which components you will be rendering in each case.

NavLink or Link is equivalent to `<a> `tag, so we only specify the url link here and Route does the job for us. So you do not need `element={<BeerDetails beer={beer} />}`